Assets/Scripts/Actors/ActorComponents/ActorPhysics.cs

```diff
@@ -179,6 +178,51 @@ public void RegisterStateMethod( ActorStates state, ActorStateMethod method )
 		}
 	}
 
+	public bool GroundedCheck()
+	{
+		_isOnGround = false;
+
+		RaycastHit hit;
+		Physics.SphereCast( transform.position + Vector3.up,
+		                    _jumpCheckRadius,
+		                    -Vector3.up * _jumpCheckDistance,
+		                    out hit,
+		                    _jumpCheckDistance,
+		                    _jumpLayer );
+		_isOnGround = ( hit.transform &&
+		                Vector3.Dot( hit.normal, Vector3.up ) > _minJumpDot );
+
+		if ( _isOnGround &&
+		     !_jumpCheck &&
+		     !IsInState( ActorStates.Climbing ) )
+		{
+			if ( !IsInState( ActorStates.Grounded ) && !IsInState( ActorStates.Rolling ) )
+			{
+				ChangeState( ActorStates.Grounded );
+				_stopMoveTimer = 0f;
+			}
+
+			actor.animator.SetBool( "isJumping", false );
+			CancelLateJumpTimer();
+			//actor.GetAnimator().SetBool("isSliding", false);
```

Comment:
The sliding stuff? @cjacobwade Are you planning on adding sliding? Or is this a remnant from pop n pals?